src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random randomFigure = new Random();
+    private final Random randomInteger = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int getRandomInt = randomInteger.nextInt(15);
+        int getRandomInt2 = randomInteger.nextInt(15);
+        int getRandomInt3 = randomInteger.nextInt(15);
+        int index = randomFigure.nextInt(Figures.values().length);
+
+        switch (Figures.values()[index].name()) {
+            case "Circle":
```

Comment:
just make the switch on random integer/enum itself, but if you only use enum for this purpose - I don't think it's necessary